src/http/services/query.service.ts

```diff
@@ -65,6 +65,17 @@ export class QueryService {
             case 'garage':
                 state.openPercent = device.state.openPercent;
                 break;
+			case 'lock':
+				state.isLocked = device.state.isLocked;
+				state.isJammed = device.state.isJammed;
+				break;
+			case 'fan':
+				state.on - device.state.on;
+				if (device.fanSpeedControl) {
+					state.currentFanSpeedSetting = device.state.currentFanSeedSetting || 'off';
```

Comment:
The state seems to be missing `currentFanSeedSetting` property. It seems to only have `availableFanSpeeds`.